config/hubs/pangeo-hubs.cluster.yaml

```diff
@@ -0,0 +1,72 @@
+name: pangeo-hubs
```

Comment:
Since this is on the *old* pangeo account, let's just use the name of the project as the name of the cluster? Otherwise we'll end up with a new pangeo cluster on the new account, and it'll get confusing. This might mean changes for the terraform workspace too, though.

Reply:
I deployed this cluster into a terraform workspace called `pangeo-test` rather than `pangeo-hubs`. I'll rename these files to `pangeo-181919`